Add Static/LiveServerTestCase for django test (#3106)
HypothesisWorks/hypothesis: hypothesis-python/RELEASE.rst

> RELEASE_TYPE: patch

Reviewer: ```suggestion
RELEASE_TYPE: minor
```

Because we're adding a new feature, this should be a `minor` release.

> This release adds `LiveServerTestCase` and `StaticLiveServerTestCase` for

Reviewer: I've just checked [the preview docs build here](https://hypothesis--3106.org.readthedocs.build/en/3106/changes.html), and it looks like we need the full cross-reference syntax:

```suggestion
This release adds :class:`~hypothesis.extra.django.LiveServerTestCase`
and :class:`~hypothesis.extra.django.StaticLiveServerTestCase` for
```

> This patch add `LiveServerTestCase` and `StaticLiveServerTestCase` for django
> test.

Reviewer: ```suggestion
This release adds `LiveServerTestCase` and `StaticLiveServerTestCase` classes for django
tests.  Thanks to Ivan Tham for this feature!
```

And please also add yourself to the alphabetized list of contributors in `AUTHORS.rst` file!
